scheduler/management/commands/trigger_periodic_task.py

```diff
@@ -73,7 +73,7 @@ def confirm(prompt):
                 model_entry.args, model_entry.kwargs,
                 **model_entry.options)
         else:
-            app.send_task(
+            async_result = app.send_task(
```

Comment:
This was unrelated to this PR, it's a bug fix from last week's hacking in Amsterdam to do a manual periodic task trigger